docs/arch.md

```diff
@@ -1,2 +1,13 @@
 # Architecture
-![kusk-gateway arch diagram](img/arch.png)
\ No newline at end of file
+
+Under the hood, Kusk Gateway consists of an Operator (Kusk Gateway Manager) and created by it K8s resources.
```

Comment:
`created by it K8s resources` not really sure what this means?

Do you mean the operator is created by K8s resources?

Reply:
created by the operator k8s resources

Reply:
suggestions for the better phrasing are welcome :)

Reply:
```suggestion
Under the hood, Kusk Gateway consists of an Operator (Kusk Gateway Manager), created by K8s resources.
```